krux/cli.py

```diff
@@ -171,6 +171,9 @@ def _init_logging(self, logger, syslog_facility, log_to_stdout):
             handler.setFormatter(formatter)
             self.logger.addHandler(handler)
 
+    def set_version(self, version):
```

Comment:
Please document with a docstring. This is especially important for abstract methods IMO.